### PR TITLE
merge freezer issue modification

### PR DESCRIPTION
**Pull request by changev**
@iceiilin @PengTian0 @panpan0000 

manifest.json doesn't have on-build-config repo

### CODE REVIEW

**Review by iceiilin — APPROVED** [review]

File: jobs/MasterCI/MasterCI [hunk 1]
```diff
@@ -66,7 +66,9 @@ node{
             }
         } catch(error){
             // trigger freeze procedure
-            build job: env.MERGE_FREEZER_TRIGGER, parameters: [[$class: 'StringParameterValue', name: 'MANUAL', value:"false"],[$class: 'BooleanParameterValue', name: 'FREEZE_OR_UNFREEZE', value:true]]
+            if (currentBuild.result == "FAILURE"){
+                build job: env.MERGE_FREEZER_TRIGGER, parameters: [[$class: 'StringParameterValue', name: 'MANUAL', value:"false"],[$class: 'BooleanParameterValue', name: 'FREEZE_OR_UNFREEZE', value:true]]
```
Comment (panpan0000) [inline jobs/MasterCI/MasterCI]: why Add a if?
when come to catch(error),  error already happened ?

Comment (iceiilin) [reply jobs/MasterCI/MasterCI]: It is to avoid merge freeze when only publish phase failed but master ci is marked as green

Comment (panpan0000) [reply jobs/MasterCI/MasterCI]: :+1

Comment (panpan0000) [reply jobs/MasterCI/MasterCI]: I think it will conflict with @PengTian0's change now.
@changev  will have to rebase

**Review by panpan0000 — APPROVED** [review]

**Pull request merged**